src/types/onyx/IntroSelected.ts

```diff
@@ -0,0 +1,6 @@
+type IntroSelected = {
+    /** The choice that the user selected in the engagement modal */
+    choice: string;
```

Comment:
Furthermore, maybe we should consider simplifying further by updating the `IntroChoices` Onyx value to be just a string rather than an object w/ single value `choice`:

```diff
diff --git a/src/types/onyx/IntroSelected.ts b/src/types/onyx/IntroSelected.ts
index f7446cdd7a..016c21953f 100644
--- a/src/types/onyx/IntroSelected.ts
+++ b/src/types/onyx/IntroSelected.ts
@@ -1,6 +1,7 @@
-type IntroSelected = {
-    /** The choice that the user selected in the engagement modal */
-    choice: string;
-};
+import type {ValueOf} from 'type-fest';
+import type CONST from '@src/CONST';
+
+/** The choice that the user selected in the engagement modal */
+type IntroSelected = ValueOf<typeof CONST.INTRO_CHOICES>;
 
 export default IntroSelected;

```

Reply:
I added the const logic which I think is more maintainable and a great suggestion. 

WRT changing introSelected from being an object to a string, we can't do that because the same NVP is being saved as an object by OldDot currently, so we already have data in that format in the DB